From: Jakub Wilk <jwilk@debian.org>
Date: Thu, 8 Oct 2015 11:57:02 -0700
Subject: Update license information for IE PNG Alpha Fix.

Upstream author of IE PNG Alpha Fix agreed that every version of his script
can be distributed under the terms of the LGPL 2.1 or later version.

Bug: http://sourceforge.net/p/docutils/bugs/169/
Bug-Debian: http://bugs.debian.org/563696
Forwarded: no
Last-Update: 2011-03-02
---
 docutils/writers/s5_html/themes/default/iepngfix.htc | 2 +-
 test/functional/expected/ui/default/iepngfix.htc     | 2 +-
 test/functional/expected/ui/small-black/iepngfix.htc | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/docutils/writers/s5_html/themes/default/iepngfix.htc b/docutils/writers/s5_html/themes/default/iepngfix.htc
index 9f3d628..701260e 100644
--- a/docutils/writers/s5_html/themes/default/iepngfix.htc
+++ b/docutils/writers/s5_html/themes/default/iepngfix.htc
@@ -4,7 +4,7 @@
 <script>
 
 // IE5.5+ PNG Alpha Fix v1.0 by Angus Turnbull http://www.twinhelix.com
-// Free usage permitted as long as this notice remains intact.
+// This is licensed under the GNU LGPL, version 2.1 or later. 
 
 // This must be a path to a blank image. That's all the configuration you need here.
 var blankImg = 'ui/default/blank.gif';
diff --git a/test/functional/expected/ui/default/iepngfix.htc b/test/functional/expected/ui/default/iepngfix.htc
index 9f3d628..701260e 100644
--- a/test/functional/expected/ui/default/iepngfix.htc
+++ b/test/functional/expected/ui/default/iepngfix.htc
@@ -4,7 +4,7 @@
 <script>
 
 // IE5.5+ PNG Alpha Fix v1.0 by Angus Turnbull http://www.twinhelix.com
-// Free usage permitted as long as this notice remains intact.
+// This is licensed under the GNU LGPL, version 2.1 or later. 
 
 // This must be a path to a blank image. That's all the configuration you need here.
 var blankImg = 'ui/default/blank.gif';
diff --git a/test/functional/expected/ui/small-black/iepngfix.htc b/test/functional/expected/ui/small-black/iepngfix.htc
index fa9f996..63e6b78 100644
--- a/test/functional/expected/ui/small-black/iepngfix.htc
+++ b/test/functional/expected/ui/small-black/iepngfix.htc
@@ -4,7 +4,7 @@
 <script>
 
 // IE5.5+ PNG Alpha Fix v1.0 by Angus Turnbull http://www.twinhelix.com
-// Free usage permitted as long as this notice remains intact.
+// This is licensed under the GNU LGPL, version 2.1 or later. 
 
 // This must be a path to a blank image. That's all the configuration you need here.
 var blankImg = 'ui/small-black/blank.gif';
